test/hotspot/jtreg/vmTestbase/nsk/jvmti/scenarios/contention/TC03/tc03t002/tc03t002.cpp

```diff
@@ -45,6 +45,9 @@ static threadDesc *threadList = nullptr;
 static jint threads_count = 0;
 static int numberOfDeadlocks = 0;
 
+static const char* thread_name_prefix = "Debugee Thread";
+static size_t thread_name_prefix_len = strlen(thread_name_prefix);
```

Comment:
Nit: I'd suggest to add a `const` specifier as it is a constant.
Also, both identifiers `thread_name_prefix` and `thread_name_prefix_len` is better to capitalize to reflect they are non-mutable.